src/js/profile/indicator.js

```diff
@@ -67,4 +63,32 @@ export class Indicator extends Component {
 
         }
     }
+
+    addIndicatorText(wrapper, title, indicatorData) {
```

Comment:
I'm not convinced that this is the correct approach. A more OO way of doing it would be to create an `Indicator` parent class and then two subclasses, perhaps `TextIndicator` and `GraphIndicator`. At this point we might want to change the naming, perhaps the parent class would be `InfoBlock` and `IndicatorBlock` and `MarkdownBlock` would be the children. What do you think?

You would then need to change the rich data view code to decide which block to render. What do you think?

Reply:
Yes I agree, that way it would be easier to add and maintain new types of InfoBlocks. Pushed a commit for this suggestion. I still need to solve an issue with the positioning of the MarkdownBlock HTML on the page